src/sys/signalfd.rs

```diff
@@ -15,32 +15,25 @@
 //!
 //! Please note that signal discarding is not specific to `signalfd`, but also happens with regular
 //! signal handlers.
-use libc::{c_int, pid_t, uid_t};
+use libc;
 use unistd;
-use {Errno, Result};
-use sys::signal::signal::siginfo as signal_siginfo;
+use {Error, Errno, Result};
 pub use sys::signal::{self, SigSet};
+pub use libc::signalfd_siginfo as siginfo;
 
 use std::os::unix::io::{RawFd, AsRawFd};
 use std::mem;
 
-mod ffi {
-    use libc::c_int;
-    use sys::signal::sigset_t;
-
-    extern {
-        pub fn signalfd(fd: c_int, mask: *const sigset_t, flags: c_int) -> c_int;
-    }
-}
 
 bitflags!{
-    flags SfdFlags: c_int {
-        const SFD_NONBLOCK  = 0o00004000, // O_NONBLOCK
-        const SFD_CLOEXEC   = 0o02000000, // O_CLOEXEC
+    flags SfdFlags: libc::c_int {
```

Comment:
We do not have a convention on this yet. I would prefer module level imports, because they are shorter in general. This is a purely aesthetic argument, and thus not very strong. But I have no other arguments for or against either option.

This does, however, not have any relevance for this PR.